shell/report.c

```diff
@@ -34,35 +34,37 @@ static FileTypes file_types[] = {
     {NULL, NULL, NULL, NULL}
 };
 
+/* virtual functions */
 void report_header(ReportContext * ctx)
-{
-    ctx->header(ctx);
-}
+{ ctx->header(ctx); }
```

Comment:
Why the change in indentation?

Reply:
Those bugged me a little, and I was just trying to make that section more compact. I guess report_header(ctx) is better than using ctx->header(ctx) directly? Because of the double ctx maybe. Idk.